metabase/metabase
# Improve collection breadcrumbs in embedding

Epic: #26838
[Product doc](https://www.notion.so/metabase/Improve-collection-breadcrumbs-with-embedding-parameters-51dd704168e344158b649415fb693960)

#### How to test
1. Run Metabase with this branch as an enterprise user (port 3000)
1. Configure enterprise token
1. Enable JWT authentication and make sure you set the secret to `ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff` https://github.com/metabase/sso-examples/blob/master/app-embed-example/server.js#L9-L10
1. Clone https://github.com/metabase/sso-examples
1. Go to `/app-embed-example/`
1. Run `yarn dev` (it will start port 3001 and 3002)
1. Go to http://localhost:3001/analytics and test if embedding shows breadcrumbs as desired.
1. You could try append different parameters to see it in action e.g. http://localhost:3001/analytics?top_nav=false or http://localhost:3001/analytics?breadcrumbs=false or http://localhost:3001/analytics?breadcrumbs=false&search=true
1. Test mobile view.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/26977)
